### ISSUE TITLE
packthebag3_subir logo

### ISSUE THREAD

**Issue by ariadnalbb** [andriusmp/switchtoecodesign#5]
Subir logo o imagen en el recuadro no es fácil ni inteligible. Una vez seleccionado el archivo que se quiere subir no hay forma de terminar la tarea y volver a la página del web.

**Comment by andriusmp** [comment]
Estoy de acuerdo, no es intuitivo (debería haber algún botón de ok). He visto que el logo se carga haciendo doble clic en la imagen.
